pkg/reconciler/common/install_test.go

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	mf "github.com/manifestival/manifestival"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func setKoDataPath() (string, bool) {
+	oldKODATAPATH, found := os.LookupEnv("KO_DATA_PATH")
+	os.Setenv(KoEnvKey, filepath.Join(os.Getenv("GOPATH"), "src", "github.com", "tektoncd", "operator", "cmd", "operator", "kodata"))
```

Comment:
Any reason to not do `filepath.Join(os.Getenv("GOPATH"), "src/github.com/tektoncd/operator/cmd/operator/kodata")) ?
Also, doesn't this make the assumption that the repository is cloned in `$GOPATH/src/github.com/tektoncd/operator` ? That assumption might not be true :sweat_smile:

Reply:
Couldn't we use `testdata` for this (as below) ?

Reply:
yup, `cargo cult again`.
removed :+1: